README.md

```diff
@@ -1,9 +1,58 @@
-# Northcoders News API
+# ncNews
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+An API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+---
 
+## Getting Started
 
+### Prerequisites
 
---- 
+Before running this project, ensure you have the following installed:
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+- Node.js
+- PostgreSQL
+
+### Installation
+
+1. **Clone the repository**:
+   ```bash
+   git clone YOUR_REPO_URL_HERE
+   cd YOUR_PROJECT_DIRECTORY
+
+### Install dependencies:
+
+npm install
+
+### Set up the databases:
+
+Create two .env files in the root of your project:
+
+.env.development
+.env.test
+
+### Add the following to each file, replacing <DATABASE_NAME> with the appropriate name found in /db/setup.sql:
+
+PGDATABASE=<DATABASE_NAME>
+
+Ensure .env files are ignored by Git: The .env.* files are already included in the .gitignore file, so they will not be committed to the repository.
```

Comment:
This doesnt need to be here as you will have already checked this and this same .gitignore will be used by everyone who clones this repo down, so they wont need to check for this.